src/main/scala/mesosphere/marathon/core/launcher/impl/InstanceOpFactoryHelper.scala

```diff
@@ -78,13 +78,11 @@ class InstanceOpFactoryHelper(
     reservationLabels: ReservationLabels,
     newState: InstanceUpdateOperation.Reserve,
     resources: Seq[Mesos.Resource],
-    localVolumes: Seq[(DiskSource, LocalVolume)]): InstanceOp.ReserveAndCreateVolumes = {
+    localVolumes: Seq[(Option[String], DiskSource, LocalVolume)]): InstanceOp.ReserveAndCreateVolumes = {
```

Comment:
I would personally go with a case class here since we already have a tuple of three items and it is hard to get what is what...

Reply:
I agree, case class would be really nice here :)